### ISSUE TITLE
Cookie OffCanvas dialog does not have aria label

### ISSUE THREAD

**Issue by tobiasberge** [shopware/platform#10934]
### Technical TODO

With [7279](https://github.com/shopware/shopware/pull/7279) we introduced applying an aria-labelledby to OffCanvas menus created by `offcanvas.plugin.js`.

This is currently not used by the OffCanvas for the Cookie configurator. The should be a `data-id` added to create an aria label for screen readers.
